tests/product.py

```diff
@@ -96,5 +96,23 @@ def test_get_all_products(self):
         self.assertEqual(len(json_response), 3)
 
     # TODO: Delete product
```

Comment:
this TODO can be deleted now, right?

Reply:
Addressed this line in most recent commit